.gitignore

```diff
@@ -9,4 +9,5 @@ Cargo.lock
 params
 *.swp
 *.swo
+.vscode
```

Comment:
```suggestion
```

I wouldn't add ignores specific to your dev setup.

If you need that, do it with a global gitignore on your machine: https://stackoverflow.com/a/7335487